apps/src/code-studio/components/progress/UnitCalendarDialog.story.jsx

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import UnitCalendarDialog from './UnitCalendarDialog';
+
+const sampleLessons = [
```

Comment:
One things I'm having a little confusion around through out the PR is what Lesson, Lesson Chunk and Lesson Schedule each mean. I think in some places you have called something of this structure a LessonSchedule or a LessonChunk. I think standardizing the language would be helpful. I think my current understanding is that Lessons are what we have at the start. We break them apart into Lesson Chunks. The full list of Lesson Chunks for a Unit is the LessonSchedule. Is that right?

Reply:
That's right! The LessonSchedule is an array of arrays. Each array corresponds with a week and contains the list of lesson chunks that happen in that week.

Reply:
Ok awesome glad thats what is going on. I think this line I called out above is one example of something where the naming isn't consistent with what we just outlined I think

Reply:
oh ok, I renamed it to sampleLessonList to clarify since it's a list of full Lessons that have yet to be split into chunks